openedx/features/badging/constants.py

```diff
@@ -20,3 +20,8 @@
 INVALID_TEAM_ERROR = 'Cannot assign badge {badge_id} for invalid team {community_id}'
 TEAM_BADGE_ERROR = 'Cannot assign missing badges to user {user_id} in team {team_id}'
 UNKNOWN_COURSE_ERROR = 'Cannot assign badge {badge_id} for team {community_id} in unknown course'
+
+# philu notification type
+NOTIFICATION_TYPE = '{prefix}.{type}'
```

Comment:
Please change it to 
`EARNED_BADGE_NOTIFICATION = 'philu.badging.user-badge-earned'`
and use this to register and fire notification to avoid redundancy

Reply:
done